### PR TITLE
Fixed the second script

### PR DESCRIPTION
**Pull request by paolosalvatori**
I fixed the second script to get the principal id of the AGIC manage identity in the node resource group starting from the name and resource group of the AKS cluster.

## Checklist
- [ ] The title of the PR is clear and informative
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] Issues addressed by the PR are mentioned in the description followed by `Fixes`.

### CODE REVIEW

File: docs/features/appgw-ssl-certificate.md [hunk 1]
```diff
@@ -48,7 +48,16 @@ appgwName=""
 resgp=""
 vaultName=""
 location=""
-agicIdentityPrincipalId=""
+aksClusterName=""
+aksResourceGroupName=""
+appgwName=""
+
+# get the resource group name of the AKS cluster
+nrg=$(az aks show --name SallyAks --resource-group SallyRG --query nodeResourceGroup --output tsv)
```
Comment (zioproto) [inline docs/features/appgw-ssl-certificate.md:56]: `SallyAks` and `SallyRG` should be parameters

**Review by zioproto — CHANGES_REQUESTED** [review]

**Comment by zioproto** [comment]
Relate to issue https://github.com/Azure/application-gateway-kubernetes-ingress/issues/1156

File: docs/features/appgw-ssl-certificate.md [hunk 2]
```diff
@@ -48,7 +48,16 @@ appgwName=""
 resgp=""
 vaultName=""
 location=""
-agicIdentityPrincipalId=""
+aksClusterName=""
+aksResourceGroupName=""
+appgwName=""
+
+# get the resource group name of the AKS cluster
```
Comment (akshaysngupta) [inline docs/features/appgw-ssl-certificate.md:55]: This approach will only work for the addon. Can you please add a comment here?

Comment (paolosalvatori) [reply docs/features/appgw-ssl-certificate.md:55]: If you intend the naming convention for the AGIC managed identity, this is true. Let me add a comment.

**Review by akshaysngupta — APPROVED** [review]
thanks!

**Comment by paolosalvatori** [comment]
Thanks @akshaysngupta 😸

**Pull request merged**